### ISSUE TITLE
Unsupported operand types

### ISSUE THREAD

**Issue by wstevens** [browscap/browscap-php#21]
Fatal error: Unsupported operand types in /home/wstevens/application/libraries/phpbrowscap/Browscap.php on line 369

This is breaking on $browser += $value; inside of the getBrowser() function. Basically $value is NULL.

This is the serialized string:
'a:6:{i:3;N;i:4;s:11:"Chrome 34.0";i:10;s:4:"34.0";i:11;s:2:"34";i:21;s:4:"true";i:36;s:1:"3";}'

User agent is:
Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.131 Safari/537.36

I am only getting this error in Chrome, where the current version is 34.0.1847.131.

Things were working fine for a while, I'm not sure why I'm suddenly getting this error.

Also in Firefox 28.0, while I'm not getting this error, browser and browser version are both  empty strings. The preg_match always fails.

I tried the latest version of browscap and things are still broken.

Basically browscap has suddenly become unusable.

Kind regards
